addons/web/static/tests/_framework/async_step.js

```diff
@@ -37,7 +37,7 @@ const checkStepState = (forceVerifySteps) => {
         forceVerifySteps ||
         (expectedSteps.length === steps.length && expectedSteps.every((s, i) => s === steps[i]))
     ) {
-        expect.verifySteps(expectedSteps);
```

Comment:
Note: this broke `ignoreOrder`.
The `verifySteps` here is not checking anything it's just there to print in case of error.
The true check is done in the `if` just above.